lsyncd: add page (#10789)
tldr-pages/tldr: pages/linux/lsync.md

> # lsync

Reviewer: ```suggestion
# lsyncd
```

> > Transfer files either to or from a remote host (but not between two remote hosts), by default using SSH.
> > It is often used to keep two directories on separate systems in sync, ensuring that changes made in one directory are immediately mirrored to the other.

Reviewer: ```suggestion
> Watch files and directories and run `rsync` when they change.
> It is often used to keep two directories on separate systems in sync, ensuring that changes made in one directory are immediately mirrored to the other.
```

> More information: <https://github.com/lsyncd/lsyncd>.

> - Lsync with rsync-share ‘share’:

Reviewer 2: ```suggestion
- Lsync with `rsync-share` share:
```

> `lsyncd -rsync {{/source_directory}} {{destination_host:/destination_directory}}`

Reviewer 2: ```suggestion
`lsyncd -rsync {{/source_directory}} {{destination_host::/destination_directory}}`
```

> - Lsync with `rsync-share` share:
> 
> `lsyncd -rsync {{/source_directory}} {{destination_host::/destination_directory}}`

Reviewer: https://github.com/lsyncd/lsyncd

>Lsyncd watches a local directory trees event monitor interface (inotify or fsevents). It aggregates and combines events for a few seconds and then ***spawns one (or more) process(es) to synchronize the changes. By default this is [rsync](http://rsync.samba.org/)***.

> `lsyncd -rsync /home remotehost.org::share/`
>
> This watches and rsyncs the local directory /home with all sub-directories and transfers them to 'remotehost' using the rsync-share 'share'.

```suggestion
- Watch the source for changes and run `rsync` to synchronize files to the destination on every change: 

`lsyncd -rsync {{path/to/source}} {{host::share_name}}`
```

> - Lsync with ssh connection:

Reviewer 2: ```suggestion
- Lsync with SSH connection:
```

> - Lsync with SSH connection:
> 
> `lsyncd -rsyncssh {{/home}} {{remotehost.org}} {{backup-home/}}`

Reviewer: ```suggestion
- Use SSH instead of `rsyncd` shares:

`lsyncd -rsyncssh {{path/to/source}} {{host}} {{path/to/destination}}`
```
